modules/doobie/src/main/scala/com/evolutiongaming/smetrics/DbMetrics.scala

```diff
@@ -0,0 +1,47 @@
+package com.evolutiongaming.smetrics
+
+import cats.Monad
+import cats.effect.Resource
+import cats.implicits._
+import com.evolutiongaming.smetrics.MetricsHelper._
+
+import scala.concurrent.duration.FiniteDuration
+
+trait DbMetrics[F[_]] {
+  def query(time: FiniteDuration, success: Boolean): F[Unit]
+}
+
+object DbMetrics {
```

Comment:
as you put that into root `com.evolutiongaming.smetrics` package, I'd call it DoobieMetrics or TransactorMetrics, and avoid "generic" names

Reply:
but this trait can be used not only for doobie, maybe we can put it to `db` package?

Reply:
resting such simple thing will complicate adding new "doobie" related features if any. Not sure that worths doing that

Reply:
ok, fixed